### PR TITLE
Kratos logo is written by the Kratos Logger

### PR DESCRIPTION
**Pull request by maceligueta**

### CODE REVIEW

**Comment by jginternational** [comment]
Love ASCIIART

```

o  o o--o    O  o-O-o  o-o   o-o                                      
| /  |   |  / \   |   o   o |                                         
OO   O-Oo  o---o  |   |   |  o-o                                      
| \  |  \  |   |  |   o   o     |                                     
o  o o   o o   o  o    o-o  o--o                                      
                                                                      
                                                                      
o   o o   o o    o-O-o o-O-o o--o  o  o o   o  o-o  o-O-o   o-o  o-o  
|\ /| |   | |      |     |   |   | |  |  \ /  |       |    /    |     
| O | |   | |      |     |   O--o  O--O   O    o-o    |   O      o-o  
|   | |   | |      |     |   |     |  |   |       |   |    \        | 
o   o  o-o  O---o  o   o-O-o o     o  o   o   o--o  o-O-o   o-o o--o  
                                                                      
                                                                      

```

File: kratos/sources/kernel.cpp [hunk 1]
```diff
@@ -18,11 +18,11 @@
 namespace Kratos {
 Kernel::Kernel() : mpKratosCoreApplication(Kratos::make_shared<KratosApplication>(
                 std::string("KratosMultiphysics"))) {
-    std::cout << " |  /           |             " << std::endl;
-    std::cout << " ' /   __| _` | __|  _ \\   __|" << std::endl;
-    std::cout << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
-    std::cout << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
-    std::cout << "           Multi-Physics " << KRATOS_VERSION << std::endl;
+    KRATOS_INFO("Kratos Core Load") << " |  /           |             " << std::endl;
```
Comment (philbucher) [inline kratos/sources/kernel.cpp]: Now in every line there will be an additional `Kratos Core Load`

Maybe you save the logo in a stringstream and pass it to the logger such that the `Kratos Core Load` appears only once

Furthermore now the logo wont show up any more if the verbosity is set to a higher level (StructuralMechanics does this in the tests)

Comment (maceligueta) [reply kratos/sources/kernel.cpp]: Thanks, I'll try a couple of things and push a proposal.

**Comment by maceligueta** [comment]
@philbucher : if I use: KRATOS_INFO("") << " |  /           |             " << std::endl; (and so on)
The output looks like this:

```
:  |  /           |             
:  ' /   __| _` | __|  _ \   __|
:  . \  |   (   | |   (   |\__ \ 
: _|\_\_|  \__,_|\__|\___/ ____/
:            Multi-Physics 6.0.0-84005d623c
```
I think it is still cool.

**Comment by maceligueta** [comment]
If nobody complains, I will push the style I just talked about.

**Comment by philbucher** [comment]
Should this then also be used for the applications?

**Comment by maceligueta** [comment]
Yes, I think so. All the output should go through the same pipe, so you can redirect it, filter it, etc... including python prints, which is broken right now (the python logger).

**Comment by maceligueta** [comment]
I forgot to mention that @roigcarlo is taking care of the python logger. I think there's no PR yet.

**Comment by roigcarlo** [comment]
Yep, actually is fixed but may wild problems appeared when adding the tests, so I think I will just upload the fix and i will left the tests for another PR

**Comment by maceligueta** [comment]
At the @KratosMultiphysics/technical-committee, it was decided to remove ':' when the label is empty. I think we can use this branch to do this change.

**Comment by maceligueta** [comment]
I merged master into this branch to test the Python Logger, which was fixed by another PR, but modified as well in this one.

**Comment by maceligueta** [comment]
@roigcarlo: it works perfectly. In python, when you don't put any label, it just prints the message, no ':' and no label. The Kratos logo appears without colon as well.
I think it can be approved. 
@roigcarlo , you said you were preparing tests for this, but experiencing problems?

File: kratos/sources/kernel.cpp [hunk 2]
```diff
@@ -18,11 +18,11 @@
...
-    std::cout << " |  /           |             " << std::endl;
-    std::cout << " ' /   __| _` | __|  _ \\   __|" << std::endl;
-    std::cout << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
-    std::cout << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
-    std::cout << "           Multi-Physics " << KRATOS_VERSION << std::endl;
+    KRATOS_INFO("") << " |  /           |             " << std::endl;
+    KRATOS_INFO("") << " ' /   __| _` | __|  _ \\   __|" << std::endl;
+    KRATOS_INFO("") << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
+    KRATOS_INFO("") << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
+    KRATOS_INFO("") << "           Multi-Physics " << KRATOS_VERSION << std::endl;
```
Comment (pooyan-dadvand) [inline kratos/sources/kernel.cpp]: I know that this is not a performance bottleneck point but to make it in a good way you should avoid using KRATOS INFO macro in each line and calling it once in order to remove unnecessary performance overhead and preventing also any other message interlace your message lines.

**Comment by roigcarlo** [comment]
Tests are on their way I have the idea but didn't have time to implement it, right now are low on the priority list so expect a week or two before I can finish them (Maybe I can finish them in 10 min Idk). I think we can go ahead with this.

**Comment by maceligueta** [comment]
I just unified the whole logo in a single print.

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Pull request merged**